flight/core/Dispatcher.php

```diff
@@ -4,10 +4,11 @@
 
 namespace flight\core;
 
-use Closure;
 use Exception;
 use flight\Engine;
 use InvalidArgumentException;
+use Psr\Container\ContainerInterface;
```

Comment:
And this needs to be removed right?

Reply:
don't forget, it doesn't affect runtime at all, within flight it works because Dice also installs the interface, flight clients that don't use the interface are not affected